driver-core/src/main/com/mongodb/client/model/Aggregates.java

```diff
@@ -173,6 +174,26 @@ public static Bson unwind(final String fieldName) {
         return new BsonDocument("$unwind", new BsonString(fieldName));
```

Comment:
That isnt backwards compatible as the new method introduces a different structure, which is forwards compatible.